eip-1646.md

```diff
@@ -0,0 +1,65 @@
+---
+eip: <to be assigned>
+title: Decentralized Autonomous Zero-identity Protocol
+author:Derek Zhou(周朝晖, zhous1998@gmail.com) (@zhous); Clinton Tong(21239506@qq.com); Yuefei Tan(whtyfhas@gmail.com)</a>; 
+discussions-to: https://github.com/ethereum/EIPs/issues/1649
+status: Draft
+type: Standards Track (ERC)
+category (*only required for Standard Track): ERC
+created: 2018-11-06
+requires (*optional): https://eips.ethereum.org/EIPS/eip-191
```

Comment:
Just needed to mention the EIP numbers here. Not the links and not the word 'EIP'